src/UI/templates/default/Breadcrumbs/tpl.breadcrumbs.html

```diff
@@ -1,9 +1,11 @@
 <nav aria-label="{ARIA_LABEL}" class="breadcrumb_wrapper">
-	<div class="breadcrumb">
-		<!-- BEGIN crumbs -->
-		<span class="crumb">
-			{CRUMB}
-		</span>
-		<!-- END crumbs -->
+	<div class="breadcrumb"> <!-- this here exists to put ellipsis on the right side via rtl-rendering --!>
+		<span> <!-- this here exists to force rendering of words in ltr order... --!>
+			<!-- BEGIN crumbs -->
+			<span class="crumb">
+				{CRUMB}
+			</span>
+			<!-- END crumbs -->
+		<span>
```

Comment:
Doesn’t really matter since Safari’s behavior in 16.4 seems to have been a bug after all, but just out of curiosity: Shouldn’t this be a `</span>`?

Reply:
This is of course correct =) Thx!